lib/chef-vault/item.rb

```diff
@@ -86,18 +86,18 @@ def clients(search_or_client, action = :add)
       else
         results_returned = false
         query = Chef::Search::Query.new
-        query.search(:node, search_or_client) do |node|
+        query.search(:node, search_or_client, filter_result: { name: ["name"] }, rows: 100000) do |node|
```

Comment:
It's a bit of a magic number; it has to be set to something (otherwise we'll just get the default, which is 1000 rows), and I think if we end up returning 100k rows we'll actually want to drop that number and do paging rather than returning that much in one go. So yes, it should probably be a constant.
That said, I'm right now only testing on 250 nodes, so I'm not getting close to this. Next round will be around 1000 nodes